voluncheer/voluncheer/settings.py

```diff
@@ -34,15 +34,20 @@
 #     "voluncheer-dev.us-west-2.elasticbeanstalk.com",
 # ]
 # Security
-CSRF_TRUSTED_ORIGINS = ["http://localhost", "http://127.0.0.1"]
+CSRF_TRUSTED_ORIGINS = [
+    "http://localhost",
+    "http://127.0.0.1",
+    "http://172.31.24.255",
+    "http://44.235.25.158",
+]
 _CSRF_TRUSTED_ORIGINS_CSV = os.getenv("CSRF_TRUSTED_ORIGINS_CSV")
 if _CSRF_TRUSTED_ORIGINS_CSV:
-    CSRF_TRUSTED_ORIGINS = _CSRF_TRUSTED_ORIGINS_CSV.split(",")
+    CSRF_TRUSTED_ORIGINS.extend(_CSRF_TRUSTED_ORIGINS_CSV.split(","))
 
-ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+ALLOWED_HOSTS = ["*", "localhost", "127.0.0.1", "172.31.24.255", "44.235.25.158"]
```

Comment:
If we are using a wildcard, do we need the other IPs? Is it "safe" to use a wildcard?

Reply:
Same as above